### PR TITLE
awaitReply() fix

### PR DESCRIPTION
**Pull request by AndrewJWin**
Added "=" to line 16 char 35.

### CODE REVIEW

File: modules/functions.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = (client) => {
   
     */
   client.awaitReply = async (msg, question, limit = 60000) => {
-    const filter = m=>m.author.id = msg.author.id;
+    const filter = m=>m.author.id == msg.author.id;
```
Comment (YorkAARGH) [inline modules/functions.js]: can you please make it `===` instead of `==`
http://evie.ban-hammered.me/i/zjkqz.png

**Comment by Soumil07** [comment]
While you're at it can you space the filter function properly? Makes it more aesthetically pleasing.

**Comment by AndrewJWin** [comment]
@Soumil07 It's spaced according to the eslinter.json file provided. It is already spaced properly.

**Comment by YorkAARGH** [comment]
To be fair it's been like that from the start, if the linter isn't complaining no harm.

**Pull request merged**